src/components/Index/ContributorsSection.tsx

```diff
@@ -4,112 +4,96 @@ import {
-            <rect
-              width="404"
-              height="384"
-              fill="url(#74b3fd99-0a6f-4271-bef2-e80eeafdf357)"
-            />
-          </svg>
-        </div>
-      </div>
-      <div className="relative px-4 sm:px-6 lg:px-8">
-        <div className="text-lg max-w-prose mx-auto">
-          <p className="text-base text-center leading-6 text-blue-600 dark:text-blue-300 font-semibold tracking-wide uppercase">
-            Contributors
-          </p>
-          <h1 className="mt-2 mb-8 text-3xl text-center leading-8 font-extrabold tracking-tight text-gray-900 dark:text-gray-100 sm:text-4xl sm:leading-10">
-            By the Community, for the Community.
-          </h1>
-          <p className="text-lg text-gray-500 dark:text-gray-400 leading-8 mb-6">
-            Interested in joining the USACO Guide team? We are recruiting
-            Content Authors to help us improve the USACO Guide!{' '}
-            <a
-              href="https://docs.google.com/document/d/13xR2A2mOftVzlC6QTSkm3zLLdFtI1NhlzRWJ81FfU9U/edit"
-              target="_blank"
-              rel="noreferrer"
-              className="text-blue-600 dark:text-blue-400 font-bold"
-            >
-              Apply here.
-            </a>{' '}
-            If you're in the United States, you can earn volunteer hours too!
-          </p>
-          <p className="text-gray-500 dark:text-gray-400 mb-2">
-            Current Content Authors:
-          </p>
-          <ul className="text-gray-500 dark:text-gray-400 list-disc list-inside space-y-2 mb-6">
-            {contentAuthors.map(author => (
-              <li key={author.name}>
-                <a
-                  href={`https://github.com/${author.githubUsername}`}
-                  target="_blank"
-                  rel="noreferrer"
-                  className="text-blue-600 dark:text-blue-400"
-                >
-                  {author.name}
-                </a>{' '}
-                (since {author.sinceDate})
-              </li>
-            ))}
-            <li className="italic">
-              This could be you!{' '}
+    <div className="bg-white dark:bg-dark-surface">
+      <div className="max-w-7xl mx-auto py-12 px-4 text-center sm:px-6 lg:px-8 lg:py-24">
+        <div className="space-y-8 sm:space-y-12">
+          <div className="text-lg mx-auto">
+            <p className="text-base text-center leading-6 text-blue-600 dark:text-blue-300 font-semibold tracking-wide uppercase">
+              Contributors
+            </p>
+            <h1 className="mt-2 mb-8 text-3xl text-center leading-8 font-extrabold tracking-tight text-gray-900 dark:text-gray-100 sm:text-4xl sm:leading-10">
+              By the Community, for the Community.
+            </h1>
+            <p className="text-lg text-gray-500 dark:text-gray-400 leading-8 mb-6 mx-auto">
+              Interested in joining the USACO Guide team? We are recruiting
+              Content Authors to <br></br>
```

Comment:
why linebreak? looks weird when window is resized :/
![image](https://user-images.githubusercontent.com/63619830/127788113-2a02f8bf-92d3-4633-9216-0e56b51d7fce.png)

Reply:
😳